vm/src/lib.rs

```diff
@@ -24,13 +24,15 @@ use std::fmt::{self, Debug, Formatter};
 use std::path::{Path, PathBuf};
 use std::thread;
 
-use dusk_core::abi::{Metadata, Query};
-use piecrust::{SessionData, VM as PiecrustVM};
+use hex as _;
```

Comment:
please add this line in a unit test environment like seen in `src/execute.rs` line 238.
You can even add it to the other test imports (`ff` and `once_cell`) to have them grouped together.